docker/prestooperator.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/presto-operator', '-f', 'docker/Dockerfile', '.']
+images:
+  - 'gcr.io/$PROJECT_ID/presto-operator'
```

Comment:
Does this resolve to `gcr.io/prestodb/presto-operator`?  By the way, there is a Docker hub for Presto, should we use that?

Reply:
This is a GCP specific build file. This will create an image in GCP of one who is using it. But, yeah we need to use the presto docker hub for uploading the presto operator's image